tests/accelerators/test_common.py

```diff
@@ -22,3 +27,52 @@ def test_auto_device_count(device_count_mock):
     assert GPUAccelerator.auto_device_count() == 2
     assert TPUAccelerator.auto_device_count() == 8
     assert IPUAccelerator.auto_device_count() == 4
+
+
+def test_pluggable_accelerator(tmpdir):
+    class TestAccelerator(Accelerator):
+        @staticmethod
+        def parse_devices(devices) -> int:
+            """Accelerator Parsing logic."""
+            return devices
+
+        @staticmethod
+        def get_parallel_devices(devices):
+            return [torch.device("cpu")] * devices
+
+        @staticmethod
+        def auto_device_count() -> int:
+            """Get the devices when set to auto."""
+            return 1
+
+        @staticmethod
+        def is_available() -> bool:
+            return True
```

Comment:
```suggestion
        def parse_devices(devices):
            return devices

        @staticmethod
        def get_parallel_devices(devices):
            return [torch.device("cpu")] * devices

        @staticmethod
        def auto_device_count():
            return 1

        @staticmethod
        def is_available():
            return True
```
typing and docstrings are not required in tests.